monai/transforms/croppad/array.py

```diff
@@ -472,7 +458,7 @@ def __init__(self, roi_size: Sequence[int] | int) -> None:
         self.roi_size = roi_size
 
     def compute_slices(self, spatial_size: Sequence[int]):  # type: ignore
-        roi_size = fall_back_tuple(self.roi_size, spatial_size)
+        roi_size = fall_back_tuple(self.roi_size, convert_to_numpy(spatial_size, wrap_sequence=True))
```

Comment:
why need add `convert_to_numpy` here?

Reply:
I think I ran into a situation that spatial_size is None or an integer, maybe we can remove it for now. Thanks!